libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/headerSection.spec.ts

```diff
@@ -0,0 +1,110 @@
+// (C) 2023 GoodData Corporation
+
+import * as Navigation from "../../tools/navigation";
+import { Widget } from "../../tools/widget";
+import { EditMode } from "../../tools/editMode";
+import { InsightsCatalog } from "../../tools/insightsCatalog";
+import { LayoutRow } from "../../tools/layoutRow";
+
+const editMode = new EditMode();
+const insightCatalog = new InsightsCatalog();
+const layoutRow_01 = new LayoutRow(0);
+const layoutRow_02 = new LayoutRow(1);
+
+Cypress.Cookies.defaults({
+    preserve: ["GDCAuthTT", "GDCAuthSTT", "_csrfToken"],
+});
+
+Cypress.on("uncaught:exception", (error) => {
+    console.error("Uncaught exception cause", error);
+    return false;
+});
+
+Cypress.Cookies.debug(true);
+
+describe("Header section", { tags: ["pre-merge_isolated_tiger"] }, () => {
+    describe("Default language", () => {
+        beforeEach(() => {
+            cy.login();
+
+            Navigation.visit("dashboard/header");
+            editMode.isInEditMode();
+            insightCatalog.waitForCatalogReload();
+        });
+
+        it("(SEPARATE) can update header for all sections", () => {
+            let i = 0;
+            let t = 0;
+            cy.fixture("headerDataTest").then((data) => {
+                data["DataTest"].forEach((result: { sectionName: string }) => {
+                    new LayoutRow(i)
+                        .getHeader()
+                        .setTitle(result.sectionName)
+                        .setDescription(result.sectionName);
+                    i += 1;
+                });
+            });
+
+            editMode.save(); //save all headers after input
+
+            cy.fixture("headerDataTest").then((data) => {
+                data["DataTest"].forEach((result: { sectionName: string }) => {
+                    if (t === 0) {
+                        new LayoutRow(t).getHeader().hasTitle(false).hasDescription(false);
+                    } else {
+                        new LayoutRow(t)
+                            .getHeader()
+                            .hasTitleWithText(result.sectionName)
+                            .hasDescriptionWithText(result.sectionName);
+                    }
+                    t += 1;
+                });
+            });
+        });
```

Comment:
no need to split into 2 cy.fixture, using the first one is enough, put the body of the second one to the first one
to check case empty section, we can check `LayoutRow(t).getHeader().hasTitle(false).hasDescription(false);` first (because the default section is always empty), then after that we call cy.fixture, remove case `"sectionName": "  "` in fixture json file

Reply:
The case is that input all sections, then save the dashboard and check after saving